changelogs/fragments/727_filetree_example.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
I don't think this needs a changelog fragment, as it's a docs only change (and not fixing something that was terribly wrong).